content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -2,898 +2,998 @@
+   :align: center
 
-Debit Notes
+Debit notes
 -----------
 
-As part of the Chilean localization, besides creating credit notes from an existing document you
-can also create debit Notes. For this just use the button “Add Debit Note”. The two main use cases
-for debit notes are detailed below.
+As part of the Chilean localization, besides creating credit notes from an existing document you can
+also create debit notes. For this just use the button :guilabel:`Add Debit Note`. The two main use
+cases for debit notes are detailed below.
 
-Use Cases
+.. _chile/use-cases:
+
+Use cases
 ~~~~~~~~~
 
-Add debt on Invoices
+Add debt on invoices
 ********************
 
-The most common use case for debit notes is to increase the value of an existing invoice, you
-need to select option 3 in the field Reference code SII:
+The most common use case for debit notes is to increase the value of an existing invoice, you need
+to select option :guilabel:`3. Corrige el monto del Documento de Referencia` for the
+:guilabel:`Reference Code SII` field.
 
-.. image:: chile/Chile31.png
+.. image:: chile/debit-note-correct-amount.png
+   :alt: Debit note correcting referenced document amount.
    :align: center
-   :alt: Debit note for partial refund to crrect amounts, using the SII reference code 3.
-
 
-In this case Odoo automatically includes the source invoice in the cross reference section:
+In this case Odoo automatically includes the :guilabel:`Source Invoice` in the :guilabel:`Cross
+Reference` tab.
 
-.. image:: chile/Chile32.png
+.. image:: chile/auto-ref-debit-note.png
+   :alt: Automatic reference to invoice in a debit note.
    :align: center
-   :alt: Invoice data on crossed reference section for debit notes.
 
-Cancel Credit Notes
+.. tip::
+   You can only add debit notes to an invoice already accepted by the SII.
+
+Cancel credit notes
 *******************
 
-In Chile the debits notes are used to cancel a validated Credit Note, in this case just
-select the button Add debit note and select the first option in the wizard: *1: Anula
-Documentos de referencia.*
+In Chile, the debits notes are used to cancel a valid credit note. In this case, just select the
+:guilabel:`Add Debit Note` button and then select the :guilabel:`1: Anula Documentos de referencia`
+option for the :guilabel:`Reference Code SII` field.
 
-.. image:: chile/Chile33.png
+.. image:: chile/debit-note-cancel-ref-doc.png
+   :alt: Debit note to cancel the referenced document (credit note).
    :align: center
-   :alt: Creating a debit note to cancel a credit note with the SII code reference 1.
 
-Vendor Bills
+Vendor bills
 ------------
 
-As part of the Chilean localization, you can configure your Incoming email server as the same you
-have register in the SII in order to:
+As part of the Chilean localization, you can configure your incoming email server as the same you
+have registered in the :abbr:`SII (Servicio de Impuestos Internos)` in order to:
```

Comment:
Personally, I thought the wording "as the same" sounded off, but I can't articulate why
```suggestion
As part of the Chilean localization, you can configure your incoming email server to match the one
you have registered in the :abbr:`SII (Servicio de Impuestos Internos)` in order to:
```